test/integration/consul-container/libs/consul-node/consul-node.go

```diff
@@ -0,0 +1,17 @@
+package consul_node
+
+import "github.com/hashicorp/consul/api"
+
+// ConsulNode represent a Consul node abstraction
+type ConsulNode interface {
```

Comment:
Could we go with `Node`? We're already in the Consul repo 😉 

```suggestion
type Node interface {
```